.claude/scripts/comment-disclosure-drift-go/main.go

```
@@ -0,0 +1,459 @@
+}
+
+// Report is the outcome of classifying a comment set.
+type Report struct {
+	Findings       []Finding
+	Counts         map[Verdict]int
+	Considered     int
+	SkippedAuthors int
+}
+
+// excerpt shortens a line for single-line reporting.
+func excerpt(line string, max int) string {
```

golangci-lint (2.12.2)

[error] param max has same name as predeclared identifier

(predeclared)